files/en-us/web/api/subtlecrypto/derivebits/index.html

```diff
@@ -94,9 +94,11 @@ <h3 id="Exceptions">Exceptions</h3>
 <p>The promise is rejected when one of the following exceptions are encountered:</p>
 
 <dl>
+  <dt>{{exception("OperationError")}}</dt>
+  <dd>Raised if the <em>length</em> parameter of the <code>deriveBits()</code> call is null, and also in some cases if the <em>length</em> parameter is not a multiple of 8.</dd>
   <dt>{{exception("InvalidAccessError")}}</dt>
   <dd>Raised when the base key is not a key for the requested derivation algorithm or if
-    the {{domxref("CryptoKey.usages")}} value of that key doesn't contain
+    the <a href="http://localhost:3000/en-US/docs/Web/API/CryptoKey#usages"><code>usages</code></a> value of that key doesn't contain
```

Comment:
I think this should be this?
```suggestion
    the <a href="/en-US/docs/Web/API/CryptoKey#usages"><code>usages</code></a> value of that key doesn't contain
```

Reply:
> I think this should be this?

oops yeah — thanks